pkg/webhook/validate_networks_test.go

```diff
@@ -0,0 +1,47 @@
+package webhook
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/controllers"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/vars"
+)
+
+func TestValidate_NetworkNamespace(t *testing.T) {
+	g := NewGomegaWithT(t)
+	defer func(previous string) { vars.Namespace = previous }(vars.Namespace)
+	vars.Namespace = "operator-namespace"
+
+	validNetworkNamespaces := []controllers.NetworkCRInstance{
```

Comment:
any chance to have a "[]tcase" (list of test case struct) which contains the test name, the NetworkCRInstance and a `shouldFail` boolean ? then the test would iterate over entries , run validateNetworkNamespace and compare to expected result ?